examples/combobox/combobox-datepicker.html

```diff
@@ -0,0 +1,782 @@
+                  </ul>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </section>
+
+      <section>
+          <h3 id="rps_label_4">Date Picker Dialog: Calendar Navigation Buttons</h3>
+          <table aria-labelledby="rps_label_4 rps_label" class="data attributes">
+            <thead>
+              <tr>
+                <th scope="col">Role</th>
+                <th scope="col">Attribute</th>
+                <th scope="col">Element</th>
+                <th scope="col">Usage</th>
+              </tr>
+            </thead>
+            <tbody>
+                <tr data-test-id="calendar-navigation-button-aria-label">
+                    <td></td>
+                    <th scope="row"><code>aria-label=<q><em>String</em></q></code></th>
+                    <td><code>button</code></td>
+                    <td>
+                      Defines the accessible name of the button (e.g. "Next Year").
+                    </td>
+                </tr>
+                <tr data-test-id="calendar-navigation-aria-live">
+                    <td></td>
+                    <th scope="row"><code>aria-live=<q>polite</q></code></th>
+                    <td>
+                      <code>h2</code>
+                    </td>
+                    <td>
+                      <ul>
+                        <li>When the month and/or year changes, the content of the <code>h2</code> element is updated.</li>
+                        <li>The use of the <code>polite</code> value means the screen reader will not interrupt other speech to announce the change in month and/or year.</li>
+                      </ul>
+                    </td>
+                  </tr>
+            </tbody>
+          </table>
+        </section>
+
+        <section>
+            <h3 id="rps_label_5">Date Picker Dialog: Date Grid</h3>
+            <table aria-labelledby="rps_label_5 rps_label" class="data attributes">
+              <thead>
+                <tr>
+                  <th scope="col">Role</th>
+                  <th scope="col">Attribute</th>
+                  <th scope="col">Element</th>
+                  <th scope="col">Usage</th>
+                </tr>
+              </thead>
+              <tbody>
+                  <tr data-test-id="grid-role">
+                    <th scope="row"><code>grid</code></th>
+                    <td></td>
+                    <td>
+                      <code>table</code>
+                    </td>
+                    <td>
+                     <ul>
+                        <li>Identifies the <code>table</code> element as a <code>grid</code> widget.</li>
+                        <li>Since the <code>grid</code> role is applied to a <code>table</code> element, the <code>row</code>, <code>colheader</code>, and <code>gridcell</code> roles do not need to be specified because they are implied by <code>tr</code>, <code>th</code>, and <code>td</code> tags.</li>
```

Comment:
change colheader to columnheader
```suggestion
                        <li>Since the <code>grid</code> role is applied to a <code>table</code> element, the <code>row</code>, <code>columnheader</code>, and <code>gridcell</code> roles do not need to be specified because they are implied by <code>tr</code>, <code>th</code>, and <code>td</code> tags.</li>
```